tests/components/medcom_ble/test_config_flow.py

```diff
@@ -0,0 +1,230 @@
+"""Test the Medcom Inspector BLE config flow."""
+from unittest.mock import patch
+
+from bleak import BleakError
+from medcom_ble import MedcomBleDevice
+import pytest
+
+from homeassistant import config_entries
+from homeassistant.components.medcom_ble.const import DOMAIN
+from homeassistant.const import CONF_ADDRESS
+from homeassistant.core import HomeAssistant
+from homeassistant.data_entry_flow import FlowResultType
+
+from . import (
+    MEDCOM_DEVICE_INFO,
+    MEDCOM_SERVICE_INFO,
+    UNKNOWN_SERVICE_INFO,
+    patch_async_ble_device_from_address,
+    patch_async_setup_entry,
+    patch_medcom_ble,
+)
+
+from tests.common import MockConfigEntry
+
+
+@pytest.mark.parametrize(
+    ("medcom_service", "ble_device"),
+    [
+        (
+            MEDCOM_SERVICE_INFO,
+            MedcomBleDevice(
+                manufacturer="International Medcom",
+                model="Inspector BLE",
+                model_raw="Inspector-BLE",
+                name="Inspector BLE",
+                identifier="a0d95a570b00",
+            ),
+        )
+    ],
+)
```

Comment:
It doesn't look like you need parametrize here since you only have on set of data